Data_Files/salary.csv

```diff
@@ -0,0 +1,31 @@
+YearsExperience,Salary
+1.1,39343
+1.3,46205

```

Comment:
I am not sure what is the use of this file. Can you please explain?

Reply:
I was just trying different csv files to see if the code works..So it is just one of the input files that i was initially trying..ignore that csv